dns/dyndns/src/etc/inc/plugins.inc.d/dyndns.inc

```diff
@@ -163,6 +165,7 @@ function dyndns_configure_client($conf)
         $dnsUpdateURL = "{$conf['updateurl']}",
         $forceUpdate = $conf['force'],
         $dnsZoneID = $conf['zoneid'],
+        /* $dnsToken = $conf['token'], */
```

Comment:
Probably a comment from an earlier version of the patch? Please remove if unneeded